src/Objs/_defaultPageEditingConfig.js

```diff
@@ -46,15 +46,17 @@ export const defaultPageProperties = [
   "navigationBackgroundImageGradient",
 ];
 
-export const defaultPageTitleValidation = [
-  "title",
+export const defaultPageTitleValidations = [
```

Comment:
Let's hide implementation details what is validated:

```suggestion
export const defaultPageValidations = [
```